src/forms/renderers/OAuth.tsx

```diff
@@ -40,6 +57,13 @@ const OAuthproviderRenderer = ({
     // Fetch what we need from stores
     const endpointConfigData = useEndpointConfigStore_endpointConfig_data();
     const imageTag = useDetailsForm_connectorImage_connectorId();
+    const setCustomErrors = useEndpointConfigStore_setEndpointCustomErrors();
+
+    // Used to make a better UX for users editing a config
+    const draftId = useEditorStore_id();
+    const workflow = useEntityWorkflow();
+    const isEdit =
+        workflow === 'capture_edit' || workflow === 'materialization_edit';
```

Comment:
We now need to know the state of the overall form for OAuth :( 

Not the most elegant solution but it works well and makes the UX good.